### ISSUE TITLE
Create an Accessibility Statement for the Site

### ISSUE THREAD

**Issue by mgifford** [CivicActions/accessibility#76]
We need an accessibility statement that stacks up with what we are promoting - https://medium.com/civicactions/whitehouse-gov-makes-an-accessibility-statement-5de37580209

**Comment by lukefretwell** [comment]
@mgifford can we link to this in the site footer? If you haven't already done, I'd move this to to-do for the week.

**Comment by mgifford** [comment]
This would be great. I haven't already done it. And yes, should just be a paged linked from the footer.  Doesn't have to be too specific when we start off. I like the idea of also making sure we link to the Colophon.

**Comment by lukefretwell** [comment]
@mgifford great start! 

Feedback:

* We should be consistent with header styles (my suggestion is AP Style)
* Seems like the 'Our Process' section could be broken out more. Like sections for 'Technology,' 'Color.' It's not clear what 'Our Process' even means.
* We should label site a11y issues with an 'accessibility' label that can be linked to the filter on the page under known issues
* Feedback section: Break out into bullets (Email, Submit a GitHub issue, etc.). Would keep this simple and not use the tech jargon so much.
* In general, would try to bullet more and keep simple.

Should create issues for things like this:

> We are not currently doing readability testing on the content, and it has not been reviewed for plain language.

We're using the USWDS Jekyll theme (which probably should also be highlighted under technology) and not so much any of the colors from the tool:

> The color pallett is based on work with the USWDS Color Compare Tool. The USWDS’s Color framework provides magic numbers that allow us to quickly determine if color contrast requirements have been met.

**Comment by lukefretwell** [comment]
@mgifford it would also be great to add sections we think should be included and, even if we don't do them now (just included '(in progress)' text and link to issues queue that's it's been documented as a need.

**Comment by lukefretwell** [comment]
@mgifford updated page: https://civicactions.github.io/accessibility/accessibility
